.travis.yml

```diff
@@ -1,5 +1,6 @@
 os: osx
 osx_image: xcode10.1
+language: swift
```

Comment:
do we need this? 

i think only specifying `os:` and `osx_image:` should work, right?

Reply:
I go back and forth on this.  Sorry about that.  Ill remove it.

<img width="852" alt="image" src="https://user-images.githubusercontent.com/2734154/54678963-9d126600-4adc-11e9-95ad-ef80c9a0bfe6.png">


FYI, I found the thread about Travis CI treating ```objective-c``` and ```swift``` the same.  I'll find the thread and post it shortly.

Reply:
💯